api/models/reviews.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../../db/connection");
+
+exports.fetchReviewById = (review_id) => {
+  if (isNaN(review_id)) {
```

Comment:
Instead of rejecting in this manner PSQL should throw an error if the review_id is not a number